src/ui/EditTask.svelte

```diff
@@ -459,6 +466,13 @@
                 />
             </div>
 
+            <!-- --------------------------------------------------------------------------- -->
+            <!--  Created on  -->
+            <!-- --------------------------------------------------------------------------- -->
+            <div>
+                <span>Created on:</span>
+                <code>{@html parsedCreated}</code>
+            </div>
```

Comment:
Please could you pop in a blank line here, for visual separation and consistency... Thanks.

Reply:
@vanadium23 Actually, never mind, don't worry I'll do it...